packages/plugin-typescript/src/lib/schema.ts

```diff
@@ -0,0 +1,24 @@
+import { z } from 'zod';
+import { AUDITS, DEFAULT_TS_CONFIG } from './constants.js';
+import type { AuditSlug } from './types.js';
+
+const auditSlugs = AUDITS.map(({ slug }) => slug) as [
+  AuditSlug,
+  ...AuditSlug[],
+];
+export const typescriptPluginConfigSchema = z.object({
+  tsConfigPath: z
```

Comment:
Could be shortened to just `tsconfig`. Or `project` is sometimes used (e.g. by `tsc` or `typescript-eslint`).

Reply:
used `tsconfig`